### DISCUSSION TITLE
What am I? 🔮

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1193] [Galaxy Trivia] [answered]
I run but have no 🦵. I roar but have no 👄. I wear a belt but have no 👖. I always 🥤 but never 🍔.

**Comment by peterbaccaro** [comment, marked as answer]
You are **an engine**.

**Comment by Sajansharma0017** [comment]
An Engine.

**Comment by AradhyaDixit18** [comment]
Hey @ParanoidUser can you please put some questions in the galaxy trivia so that i can get the badge please ?
